renovate.json

```diff
@@ -0,0 +1,53 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base",
+    "schedule:daily",
+    ":dependencyDashboard",
+    ":semanticCommits",
+    ":semanticCommitScope(deps)",
+    "group:linters",
+    "group:googleapis",
+    "group:goOpenapi",
+    "group:reactMonorepo",
+    "group:reactrouterMonorepo"
+  ],
+  "packageRules": [
+    {
+      "datasources": ["docker"],
+      "updateTypes": ["patch"],
+      "groupName": "docker patch & minor updates"
+    },
+    {
+      "datasources": ["go"],
+      "updateTypes": ["patch", "minor"],
+      "groupName": "go.mod patch & minor updates"
+    },
+    {
+      "datasources": ["npm"],
+      "updateTypes": ["patch", "minor"],
+      "groupName": "npm updates"
+    },
+    {
+      "datasources": ["pip"],
+      "enabled": false
+    },
+    {
+      "depTypeList": ["go_repository"],
```

Comment:
Is it your goal to disable `bazel` updates completely? If to then `"managers": ["bazel"]` can replace the `depTypeList` rule you have

Reply:
Yes, I was searching desperately in the docs page, but couldn't find it.

It'll help a lot if the possible values for `managers`, `depTypeList`, `languages`, `datasources` were explicitly listed in the documentation.

Reply:
https://docs.renovatebot.com/modules/manager/
https://docs.renovatebot.com/modules/datasource/

Reply:
Lol, I also just saw it. I totally missed the two pages.
I think it can help a lot if the reference page have the two links.